projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon.module.ts

```diff
@@ -1,16 +1,24 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { CmsConfig, ConfigModule, I18nModule } from '@spartacus/core';
+import {
+  CmsConfig,
+  ConfigModule,
+  I18nModule,
+  FeaturesConfigModule,
+} from '@spartacus/core';
 import { IconModule } from '../../../cms-components/misc/icon/index';
 import { AppliedCouponsComponent } from './applied-coupons/applied-coupons.component';
 import { CartCouponComponent } from './cart-coupon.component';
+import { NgSelectModule } from '@ng-select/ng-select';
 
 @NgModule({
   declarations: [CartCouponComponent, AppliedCouponsComponent],
   exports: [CartCouponComponent, AppliedCouponsComponent],
   imports: [
+    FeaturesConfigModule,
     CommonModule,
+    NgSelectModule,
```

Comment:
```
features: {
        customerCoupon: '1.x',
 }
```
By refer to consignmenttracking, this config is necessary.

Reply:
no feature flag for customer coupon now, it's data driven